libbeat/docs/outputs/output-elasticsearch.asciidoc

```diff
@@ -0,0 +1,657 @@
+[[elasticsearch-output]]
```

Comment:
This file is still quite massive which makes it hard to review. Did anything change or is this simply a copy/paste of the section?

Reply:
simply a copy/paste.

Reply:
simple copy/paste with conditions removed